flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceFailoverWithWatermarkExtractor.java

```diff
@@ -48,10 +52,17 @@ public class TestIcebergSourceFailoverWithWatermarkExtractor extends TestIceberg
 
   @Override
   protected IcebergSource.Builder<RowData> sourceBuilder() {
-    return IcebergSource.<RowData>builder()
+    Configuration config = new Configuration();
+    config.setInteger(FlinkConfigOptions.SOURCE_READER_FETCH_BATCH_RECORD_COUNT, 128);
```

Comment:
Copied from the `TestIcebergSourceFailover`.
Do you happen to remember why they were set in the first place?
The tests are running without it.

Reply:
I see. this config is probably not required for `TestIcebergSourceFailover`, because only small number of records are generated per data file. This config is probably copied over unnecessarily from some specific tests like `TestArrayPoolDataIteratorBatcherRowData`.

either way is fine with me (copying it here or not).